src/lib/components/Searchbar/SearchBar.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Actions, prepareLayer, lngLat } from 'gisida';
+import './SearchBar.scss'
+
+/**
+ * To activate the search functionality add "searchBar":true and 
+ * To Change search bar color add "searchBarColor": <desired color> 
+ * on client site-config.json APP object
+*/
+
+
+const mapStateToProps = (state, ownProps) => {
+    const { APP, LOC } = state;
+    const MAP = state[ownProps.mapId] || { layers: {} };
+    return {
+      APP,
+      LOC,
+      appColor: APP.appColor,
+      searchBarColor: APP.searchBarColor,
+      preparedLayers: MAP.layers,
+      ...ownProps
+    };
+  };
+
+class SearchBar extends Component {
+  constructor(props) {
```

Comment:
@ciremusyoka could/should this component be refactored as a functional component?

Reply:
@cKellyDesign our `react-dom` version doesn't support hooks.